### ISSUE TITLE
Unexpected keyword argument 'display_GMT_offset'

### ISSUE THREAD

**Issue by natebessa** [mfogel/django-timezone-field#63]
The README has the following:

`timezone2 = TimeZoneFormField(display_GMT_offset=True) # displays like "GMT-08:00 America/Los_Angeles"` 

But _display_GMT_offset_ does not appear to be a valid argument of TimeZoneFormField. It is only defined under the model class of TimeZoneField.

Is there a way another way we can get our form field to show the GMT offset like "GMT-08:00 America/Los_Angeles" ?

**Comment by drtpotter** [comment]
Yeah, this option (display_GMT_offset=True) appears to be missing for TimeZoneFormField in the latest release. Would be great to have!

**Comment by mfogel** [comment]
Sorry for the frustration here. It appears I merged a PR without doing due diligence.

I'm planning to put together a fix this week to
- get this option to work for both model form fields and form fields
- improve implementation, performance
- switch from `GMT` to `UTC` in both the option name and what gets displayed to the user, unless somebody has a good reason to stick with `GMT`. This will likely be a backward-incompatibility, so major version of the package will get bumped upon release.

**Comment by mfogel** [comment]
This has been fixed on `master` as of 0c6a6ba. You can now do the following:

```py
class MyForm(forms.Form):
    tz = TimeZoneFormField(choices_display='WITH_GMT_OFFSET')
```

and you should get a list of choices with that display like `GMT-08:00 America/Los_Angeles`. The `choices_display` kwarg also works the same for Models and ModelForms as it does for Forms.

If any of you have time to try running the latest on `master` in your own project(s) and file any bugs or unexpected behavior you come across, that would help me out a lot! I'm planning to push a new release to pypi in a week or two, and would prefer to catch any glaring bugs before then if possible. Thanks & cheers.
